diff --git a/node_modules/react-native/jest/setup.js b/node_modules/react-native/jest/setup.js
index 912b5f7..1677700 100644
--- a/node_modules/react-native/jest/setup.js
+++ b/node_modules/react-native/jest/setup.js
@@ -17,13 +17,13 @@ jest.requireActual('@react-native/polyfills/error-guard');
 
 global.__DEV__ = true;
 
-global.performance = {
+global.performance ??= {
   now: jest.fn(Date.now),
 };
 
 global.Promise = jest.requireActual('promise');
 global.regeneratorRuntime = jest.requireActual('regenerator-runtime/runtime');
-global.window = global;
+global.window ??= global;
 
 global.requestAnimationFrame = function(callback) {
   return setTimeout(callback, 0);
